Livecheckables/aubio.rb

```diff
@@ -1,4 +1,4 @@
 class Aubio
   livecheck :url   => "https://aubio.org/pub/",
-            :regex => /href="aubio-([0-9,\.]+)\.tar/
+            :regex => /aubio-(\d+(?:\.\d+)+)\.t/
```

Comment:
```suggestion
            :regex => /href="aubio-(\d+(?:\.\d+)+)\.t/
```

The `href="` shouldn't have been removed here. As I've mentioned before when talking about index pages, we want to match the `href` attribute, not text nodes, since the text can be truncated depending on the length. `href` attributes will always contain the full text of the filename and that's why we use them.